core/src/main/resources/reference.conf

```diff
@@ -1020,8 +1020,9 @@ datastax-java-driver {
     # the package com.datastax.oss.driver.internal.core.addresstranslation.
     #
     # The driver provides the following implementations out of the box:
-    # - PassThroughAddressTranslator: returns all addresses unchanged
+    # - PassThroughAddressTranslator: returns all addresses unchanged.
```

Comment:
In regards to your question around additional documentation needed, it would be nice to add something documenting `SubnetAddressTranslator` to the [Address resolution docs](https://github.com/apache/cassandra-java-driver/tree/4.x/manual/core/address_resolution) below the `EC2 Multi region` section.  Could be good to explain the use case for why this may be considered. If you are feeling adventurous it'd be nice to add a blurb on `FixedHostNameAddressTranslator` as well (I could also make an attempt at this later)